### DISCUSSION TITLE
I'm so lost without the old forum

### DISCUSSION THREAD

**Discussion by jt4242** [OpenSageTV/Community#94] [FAQ] [answered]
I had an entry bookmarked about how to fix a spinning circle in Sage but, of course,  now I can't get to it.  I did download the old forum but can't find anything in a text search of that one either.  I'm also lost in this new forum - i.e. there used to be people I wrote but messages seem gone so I can't find them either.  Even writing this took me a long time!  Any help would be greatly appreciated - such as, is there messaging with the other members anymore?  How to find an old entry in the downloaded files on my hard drive.  Sorry to be so obscure but I'm truly lost.
Julie

**Comment by Sagetvmaster** [comment]
I hear you - I had a problem this week that would have been so easy to fix if the old forum was "searchable".  As it is now, that's not an option.  And yes, it seemed more people jumped in to help a request with the old forum as well.  I think a lot of Sage users haven't been able to make this Github thing work for them, or maybe they don't know how to have messages pushed to them so they see problem requests when they pop up.  Hang in there - and ask more than once if need be.  You could always pay/subscribe to Josh's Patreon page for a spell - maybe he helps paying customers with more than just the Media Client he supports - it's worth asking him I suppose

**Reply by jt4242** [reply]
Thanks for your reply!!  It seems almost like the old days and that was the best part of the forum - one felt the whole SageTV world was behind you - very comforting.  I actually did used to contribute to Josh's Patreon page (I think - I know it was someone's patreon page but my memory has said goodbye lately!   I stopped paying awhile ago and am not sure how to sign up to it again!  It would be so nice if google would host it all again!

**Reply by emveepee** [reply]
There are also posts here on how to convert a booked mark link to get the new link.

**Reply by eelpout** [reply]
you can still search the old site using Google, but since the preprocessor no longer works there, the results have to be fixed for static pages. see here: https://github.com/OpenSageTV/Community/discussions/58#discussioncomment-6661938

I thought this was going to be fixed at Google, with a reindex of only the static pages, but so far nothing has happened. Unfortunately, only the site owner can ask for the indexing AFAIK.

and I agree, hosting a forum here is less than great. :(

**Reply by wayner9** [reply]
I agree, it is so freaking painful to search in the old forums.  And the old community is gone as these new forums don't works as well as general discussion forums.

**Reply by jt4242** [reply]
I did try converting the bookmark link 
https://forums.sagetv.com/forums/showthread.php?t=63220
to 
https://forums.sagetv.com/forums/showthread_t_620663.html 

but I just get the same 404 error.
The requested URL /forums/showthread_t_620663.html was not found on this server. That’s all we know.

**Reply by wayner9** [reply]
Actually your thread number changed from the first link to the second - this link should work:  https://forums.sagetv.com/forums/showthread_t_63220.html

**Reply by Sagetvmaster** [reply]
Agreed - I had hoped Jeff would have found a way to make the old site searchable somehow, but that hasn't happened.  It's times like this I really wish we could afford to PAY a nominal fee and have the old site back with search...

**Reply by jt4242** [reply]
I'd be ready to contribute to a fund to remount it if that were possible??  I had so much help from the forum just by reading it!!  And I always read the daily new posts.  I was so used to the format as well.

**Comment by jt4242** [comment]
Thank you!!!  I can't believe I did that - That was actually another bookmark
php?p=620663#post620663
that gave me the error - probably because of the post #??  Now does this only work if one actually has a bookmark for the item - I only have a couple.

Thank you again so much
julie

**Reply by eelpout** [reply]
are you sure there isn't an extra 0 in there? 620k+ is _a lot_ of posts and I think 10x what is actually there. Maybe it's:

https://forums.sagetv.com/forums/showthread_t_62663.html

?

**Reply by jt4242** [reply]
I'm sure - and here is yet another link with a higher post number
https://forums.sagetv.com/forums/showthread.php?p=620663#post6206630
I don't know how to convert one that has the thread number and the post number.

**Comment by emveepee** [comment, marked as answer]
There is no php so no post based links, but pages can be bookmarked.

**Comment by TGsell1** [comment]
I would also contribute.  Sent from my iPhoneOn Nov 13, 2023, at 12:46 PM, jt4242 ***@***.***> wrote:﻿
I'd be ready to contribute to a fund to remount it if that were possible??  I had so much help from the forum just by reading it!!  And I always read the daily new posts.  I was so used to the format as well.

**Comment by emveepee** [comment]
SageTV support in general is really about existing users to communicating existing users about old or ongoing problems. This forum isn't bad for that.  The best forum software in the world wouldn't fix the fundamental problem of using abandonware.

**Reply by wayner9** [reply]
Not being able to effectively search the old forums makes it hard to find hard to solve problems, and it sure seems like a lot of folks didn't bother to come to the new forums.  So those two issues will/have accelerate the decline of SageTV.

**Reply by jt4242** [reply]
Yes and I'm sure lots of us knew just where to find the answers to recurring questions.  I did try downloading the forum but somehow the file with the extension whtt was always empty - I never did get it all working and now can't remember how I'm supposed to set it up to search it.  I'm glad this forum is here but do regret the loss of all the previous data.

**Reply by jwittkoski** [reply]
The thing that most confuses me is that the new static site doesn't even seem to be fully indexed by Google. When I do a Google search limiting results using "site:" and using names/words from a post on the old site things often are not listed. Having Google index all the static pages would be very helpful since there is no local search any longer.

**Comment by JustFred51** [comment]
What will it take to get Google to index the old forums?  My attempts to search based on old topic _**titles**_ have all failed.

**Comment by KryptoNyte-Sage** [comment]
Test1

**Comment by KryptoNyte-Sage** [comment]
I don't know if this circumvents the forum email notifications, but I would suggest that when you reply to someone in this forum that you do not reply to their post (at the very bottom of their specific post), but you go to the bottom of the thread and "Suggest an answer" so it always bumps the thread to the top and marks it in bold as 'unread.'

Thoughts?

**Comment by TGsell1** [comment]
I’ll do that from now on.  I’ve been responding via email on phone ‘reply to all’ but many times the previous message doesn’t carry along.  So,,, will work off GitHub
